src/AutoRest.CSharp/Mgmt/Output/ResourceOperation.cs

```diff
@@ -1,15 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System;
-using System.Collections.Generic;
 using System.Text;
 using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Mgmt.AutoRest;
 using AutoRest.CSharp.Output.Builders;
 using AutoRest.CSharp.Output.Models.Types;
-using AutoRest.CSharp.Utilities;
 
-namespace AutoRest.CSharp.Output.Models
+#nullable enable
```

Comment:
You shouldn't need this. Nullable is enabled project-wide https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/AutoRest.CSharp.csproj#L8

Reply:
Mmm, it didn't kick in my IDE. Let me check again.

Reply:
It may be an IDE issue as ? still get red squiggles in editor but didn't affect build. I removed this from each file level.